09_simulations/nn_models.py

```diff
@@ -1,6 +1,10 @@
+import numpy as np
+import torch
 import torch.nn as nn
 import torch.nn.functional as F
 
+from skorch import NeuralNet
```

Comment:
I'm sure it's not, but that package name just looks like a typo.

Reply:
Yeah, I think it's supposed to be a merger/portmanteau between `sklearn` and `torch` (since it's an sklearn wrapper for training pytorch models) but it is a bit of a weird name. 🤷